### ISSUE TITLE
All messages on the left

### ISSUE THREAD

**Issue by Koshar** [telegramdesktop/tdesktop#1629]
Could you please bring old message layout back.
Or at least provide an option to switch it back.

**Comment by FlyingPumba** [comment]
Happening the same here. Please, fix or provide an option to switch back.

OS: Ubuntu 14.04
Telegram version: 0.9.24
Screenshot:
![screenshot blur](https://cloud.githubusercontent.com/assets/1159078/13097729/86e9013e-d502-11e5-87f8-fbb359ae8b9c.png)

**Comment by Koshar** [comment]
I just installed old version 0.9.18 and disabled auto update.

**Comment by Mindstormer619** [comment]
+1. So many +1s.

I really hope this is a bug and not a result of intentional design. It's really hard to distinguish chats that are sent by us vs those sent by the other person.

For the rest of you, there's a small fix that you can do for now. Just don't use the app in fully maximized mode. Any other window size will show the chats normally.

**Comment by diazbastian** [comment]
Please guys, this is a big improvement.

The previous design in widescreen was really inappropriate, he had much wasted space and forcing the eye to move from one place to another at random. In the new design, profile pictures added to the color of the bubbles easily distinguish who the message is from. I think this is an appropriate design for a desktop application and should be closer to something like Webogram.

Remember that you can still use the compact mode.

Cheers

**Comment by Shadorino** [comment]
I confirm this. It seems to be a bug, not a new choice in presentation

**Comment by DaFri-Nochiterov** [comment]
Option in the settings is good solution for that.

Messages style:
- [ ] One side
- [x] Centred
- [ ] Old style

**Comment by Mindstormer619** [comment]
@diazbastian 

> he had much wasted space

Wasted space? In this new presentation, _all_ the space on the right side of the screen is wasted.

> profile pictures added to the color of the bubbles easily distinguish who the message is from

The bubble color is a _really_ subtle indicator. It's white vs really light green. It's not enough to distinguish messages. Also, 1. Seeing your own profile image is not really necessary for each chat you send and 2. Many people don't _have_ a profile pic set in the first place, which is quite confusing in larger group chats.

In any case, if this was an intentional change and not just a bug as many seem to guess, the only acceptable way for this to work out is for the users to have options to select which layout they prefer.

**Comment by set92** [comment]
+1, I think we all want the old way, or at least a setting to change it.

**Comment by auchri** [comment]
#1587

**Comment by Shadorino** [comment]
So it's not a bug...
Well I agree that we should at least have the choice, like DaFri-Nochiterov said.

**Comment by Igetin** [comment]
Because people seem to have very differing preferences about this, a toggle for different layouts would seem to be the best choice here.

**Comment by marcovr** [comment]
Just add the option to switch back and everyone is happy.
Personally I really hate this crumped alignement.

**Comment by diazbastian** [comment]
@Mindstormer619

**Comment by FlyingPumba** [comment]
@diazbastian Do you know how many people DOES NOT have 4K screens ? Guess what, a large number. They should put it back as it was before, or at least add a setting option.

**Comment by diazbastian** [comment]
@FlyingPumba I said most of the cases. 1366x768 screens already exist waste of space in the center which has now moved to the right. I think the bubble system works well on mobile, but move it to the desktop is not functional. Have you used IRC or other IM system ?. Most are aligned to the left and did not exist before these complaints.

Cheers

**Comment by FlyingPumba** [comment]
@diazbastian I constantly use Telegram desktop in a 1920x1080 scren and I have never, ever, felt that the space in the center was being "wasted", because the messages in both sides balanced the "weight" in the screen. With the new layout, all the stuff is concentrated in the left side, which gives the right side a feel of "wasted", which looks horrible.
Also, if you are so concerned about the space in large displays, you can always modify the zoom setting, which is already there.
In short, in my opinion, there isn't one good thing about the new layout.

**Comment by diazbastian** [comment]
@FlyingPumba Ok, is your opinion based on aesthetics rather than functionality. Zoom options other purpose.
Discussion of this topic has been moved to another thread, maybe you can bring your arguments there.

Cheers

**Comment by telegramdesktop** [comment]
Please check out the latest 0.9.25 dev version from the bottom of this page:

https://desktop.telegram.org/#dev-version

It adds a checkbox "Adaptive layout for wide screens" below "Tile background" checkbox that can disable the new layout. It is visible only if you have a large window (with the new layout) already.

**Comment by EpicLPer** [comment]
The option is back guys <3

**Comment by Koshar** [comment]
Yay, we have won!

**Comment by colinbr96** [comment]
In case anyone can't find the setting, make sure you fullscreen Telegram. It hides itself when the screen size is small.

**Comment by lalejand** [comment]
I'm using Telegram 1.9.19 beta downloaded from the website, and I can't find the option.

**Comment by sebastianolafhandjojo** [comment]
April 2020 the chat still on the left side,  (telegram desktop)

**Comment by set92** [comment]
You can reduce the size of the window or go to settings - chat settings - disable Adaptive layout for wide screen on Chat background.

**Comment by sebastianolafhandjojo** [comment]
Thank you

**Comment by danlugo92** [comment]
Someone still getting this bug, trick to solve it:

Settings -> untoggle default interface scale -> setting it to 125% fixed it for me.

I think it somehow got auto-changed to 100% via dual-display shenanigans.

**Comment by yashpalgoyal1304** [comment]
> make sure you fullscreen Telegram. It hides itself when the screen size is small.

well, that's really bad. it should be greyed out or smth. hiding stuff is not good. i did NOT expect this.
